### PR TITLE
Docs: Update Brace example

### PR DESCRIPTION
**Pull request by kolibril13**
added a new brace example:
image

### CODE REVIEW

**Comment by hydrobeam** [comment]
I feel like that many braces is a little visually cluttering. If the goal is to show off `sharpness`, then maybe a brace with very low sharpness next to a brace with high sharpness  (on the other side of the shape)  would be more effective?

File: manim/mobject/svg/brace.py [hunk 1]
```diff
@@ -37,14 +37,16 @@ class Brace(SVGPathMobject):
...
         class BraceExample(Scene):
             def construct(self):
-                circle = Circle()
-                brace = Brace(circle, direction=RIGHT)
-                self.play(Create(circle))
-                self.play(Create(brace))
-                self.wait(2)
+                s= Square()
+                self.add(s)
+                for i in np.linspace(0.1,1.0,10):
```
Comment (kolibril13) [inline manim/mobject/svg/brace.py:46]:
```suggestion
                for i in np.linspace(0.1,1.0,4):
```

File: manim/mobject/svg/brace.py [hunk 2]
```diff
@@ -37,14 +37,16 @@ class Brace(SVGPathMobject):
...
+                    br = Brace(s, sharpness=i)
+                    self.add(br)
+                VGroup(*self.mobjects).arrange(DOWN, buff=0.1)
```
Comment (kolibril13) [inline manim/mobject/svg/brace.py:49]:
```suggestion
                VGroup(*self.mobjects).arrange(DOWN, buff=0.2)
```

**Comment by AntonBallmaier** [comment]
I like the new example :) I think it would be nice to add labels to the braces displaying the sharpness value

File: manim/mobject/svg/brace.py [hunk 3]
```diff
@@ -37,14 +37,16 @@ class Brace(SVGPathMobject):
...
             def construct(self):
-                circle = Circle()
-                brace = Brace(circle, direction=RIGHT)
-                self.play(Create(circle))
-                self.play(Create(brace))
-                self.wait(2)
+                s= Square()
+                self.add(s)
+                for i in np.linspace(0.1,1.0,4):
+                    br = Brace(s, sharpness=i)
```
Comment (kolibril13) [inline manim/mobject/svg/brace.py:47]:
```suggestion
                    br = Brace(s, sharpness=i)
                    t =Text(f"sharpness= {i}").next_to(br, RIGHT)
                    self.add(t)
```

File: manim/mobject/svg/brace.py [hunk 4]
```diff
@@ -37,14 +37,18 @@ class Brace(SVGPathMobject):
...
+                    t =Text(f"sharpness= {i}").next_to(br, RIGHT)
```
Comment (kolibril13) [inline manim/mobject/svg/brace.py:48]:
```suggestion
                    t = Text(f"sharpness= {i}").next_to(br, RIGHT)
```

**Comment by kolibril13** [comment]
image

File: manim/mobject/svg/brace.py [hunk 5]
```diff
@@ -37,14 +37,18 @@ class Brace(SVGPathMobject):
...
+        :save_last_frame:
 
         class BraceExample(Scene):
             def construct(self):
-                circle = Circle()
-                brace = Brace(circle, direction=RIGHT)
-                self.play(Create(circle))
-                self.play(Create(brace))
-                self.wait(2)
+                s= Square()
```
Comment (jsonvillanueva) [inline manim/mobject/svg/brace.py:44]:
```suggestion
                s = Square()
```

**Review by jsonvillanueva — APPROVED** [review]

**Pull request merged**